CHANGELOG.md

```diff
@@ -913,7 +913,9 @@
 - Legacy version of report csv download is available whilst the new version is
   validated by BEIS
 
-[unreleased]: https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/compare/release-86...HEAD
+## [unreleased]
+
+[release-86]: https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/compare/release-86...release-87
```

Comment:
I think the numbering got a bit mangled?

- the link to release 87 isn't present (there are 2 for 86)
-  the link to `[unreleased]: https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/compare/release-87...HEAD` isn't present

Reply:
Well that is just wrong! Thank you.